packages/ra-ui-materialui/src/AdminUI.tsx

```diff
@@ -22,7 +22,7 @@ export const AdminUI = ({
     error = Error,
     ...props
 }: AdminUIProps) => (
-    <ScopedCssBaseline enableColorScheme>
+    <ScopedCssBaseline enableColorScheme sx={{ height: '100vh' }}>
```

Comment:
Given that `DefaultLayout` is already `100vh`, you could set `margin: 'auto` to the `Root` error component instead of changing the whole app height

In `Error.tsx`:
```ts
const Root = styled('div', {
    name: PREFIX,
    overridesResolver: (props, styles) => styles.root,
})(({ theme }) => ({
    margin: 'auto', // <- this
    display: 'flex',
```
This won't work for the `Default Error` story because it doesn't have a parent with a full height. But it should work for others.